### PR TITLE
doc: gsg_guides: remove nRF52 Series and nRF5340 DK guides

### PR DESCRIPTION
**Pull request by greg-fer**
Removed the nRF52 Series and the nRF5340 DK Getting Started Guides.
The guides are now replaced by the Quick Start app.

### CODE REVIEW

File: doc/nrf/gsg_guides.rst [hunk 1]
```diff
@@ -14,9 +14,13 @@ They provide a guided experience using Nordic Semiconductor tools and precompile
 This is not a comprehensive section that includes all of the available getting started guides for Nordic Semiconductor devices.
 
 .. note::
-   * To get started with the nRF91x1 DKs and the nRF54L15 PDK, complete the steps in the `Quick Start`_ app, available from `nRF Connect for Desktop`_.
-   * To get started with the nRF54H20 DK, the installation of the |NCS| is required.
-     See :ref:`ug_nrf54h20_gs`.
+
+   The guides in the getting started section are being gradually replaced by the `Quick Start`_ app, available from `nRF Connect for Desktop`_.
+   The Quick Start app currently supports nRF52 DK, nRF52833 DK, nRF52840 DK, nRF91x1 DKs, and the nRF54L15 PDK.
```
Comment (wiba-nordic) [inline doc/nrf/gsg_guides.rst:19]:
```suggestion
   The Quick Start app currently supports the nRF52 DK, nRF52833 DK, nRF52840 DK, nRF91x1 DKs, and nRF54L15 PDK.
```
Depending on timing, PDK might need to be changed to DK.

File: doc/nrf/gsg_guides/gsg_other.rst [hunk 2]
```diff
@@ -1,11 +1,14 @@
 .. _gsg_other:
+.. _ug_nrf52_gs:
 
 Getting started with other DKs
 ##############################
 
-Use the following guides to familiarize yourself with the |NCS| tools and components for the following devices supported by the SDK:
+The guides in the getting started section are being gradually replaced by the `Quick Start`_ app, available from `nRF Connect for Desktop`_.
+The Quick Start app currently supports nRF52 DK, nRF52833 DK, nRF52840 DK, nRF91x1 DKs, and the nRF54L15 PDK.
```
Comment (wiba-nordic) [inline doc/nrf/gsg_guides/gsg_other.rst:8]:
```suggestion
The Quick Start app currently supports the nRF52 DK, nRF52833 DK, nRF52840 DK, nRF91x1 DKs, and nRF54L15 PDK.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by greg-fer** [comment]
Discussed this with @divipillai and @OisinFitz at a meeting on Friday. This PR will rework the getting started landing page into a table with the full overview of boards that have getting started guides or quick start support. A follow-up [task ](https://nordicsemi.atlassian.net/browse/NCSDK-29643) will move Thingy getting started guides from this section.

File: doc/nrf/gsg_guides/gsg_other.rst [hunk 3]
```diff
@@ -1,11 +1,9 @@
 .. _gsg_other:
 
+
```
Comment (wiba-nordic) [inline doc/nrf/gsg_guides/gsg_other.rst:3]: I don't think this page serves a purpose with this content. It'd probably be better to just add a sentence or two to the Quick Start page to point out that these devices don't have support for Quick Start.

Comment (greg-fer) [reply doc/nrf/gsg_guides/gsg_other.rst:3]: It was added by @FrancescoSer on request from @shanthanordic for those who did not click the top page and just looked at the hierarchy: 
image

I'd like to remove this page once the subpages are removed from the section.

Comment (wiba-nordic) [reply doc/nrf/gsg_guides/gsg_other.rst:3]: Alright, that makes sense.

**Review by gmarull — APPROVED** [review]

**Review by boundlesscalm — APPROVED** [review]
LGTM

File: doc/_utils/redirects.py [hunk 4]
```diff
@@ -32,13 +32,15 @@
...
-    ("device_guides/working_with_nrf/nrf53/nrf5340_gs", "gsg_guides/nrf5340_gs"), # Getting started with nRF5340 DK
+    ("device_guides/working_with_nrf/nrf53/nrf5340_gs", "gsg_guides"), # Getting started with nRF5340 DK (removed in v2.8.0)
+    ("gsg_guides/nrf5340_gs", "gsg_guides"),
     ("ug_thingy53_gs", "gsg_guides/thingy53_gs"), # Getting started with Thingy:53
     ("working_with_nrf/nrf53/thingy53_gs", "gsg_guides/thingy53_gs"),
     ("device_guides/working_with_nrf/nrf53/thingy53_gs", "gsg_guides/thingy53_gs"),
-    ("ug_nrf52_gs", "gsg_guides/nrf52_gs"), # Getting started with nRF52 Series
-    ("working_with_nrf/nrf52/gs", "gsg_guides/nrf52_gs"),
-    ("device_guides/working_with_nrf/nrf52/gs", "gsg_guides/nrf52_gs"),
+    ("ug_nrf52_gs", "gsg_guides/nrf52_gs"), # Getting started with nRF52 Series (removed in v2.8.0)
```
Comment (divipillai) [inline doc/_utils/redirects.py:40]:
```suggestion
    ("ug_nrf52_gs", "gsg_guides"), # Getting started with nRF52 Series (removed in v2.8.0)
```

File: doc/nrf/gsg_guides.rst [hunk 5]
```diff
@@ -1,24 +1,52 @@
...
-   * To get started with the nPM1300 EK, see the `Get started <nPM1300 EK get started_>`_ section of the nPM1300 EK product page.
-   * To get started with the nRF7002 EB, see the `Get started <nRF7002 EB get started_>`_ section of the nRF7002 EB product page.
+.. important::
+    The following table does not list all devices supported in the |NCS|.
+    For the full list, see the :ref:`board support pages <programming_board_names>`.
+
++----------------------+----------------------------------------------------------------+--------------------------------+
+| Device               | Getting started method                                         | Device user guide              |
++======================+================================================================+================================+
+| nRF9161 DK           |                                                                | `nRF9161 DK Hardware`_         |
```
Comment (divipillai) [inline doc/nrf/gsg_guides.rst:20]: Shouldn't we also link the PS along with the hardware guide?

**Review by divipillai — COMMENTED** [review]
The images those are only used in the GsGs could also be removed.

File: doc/nrf/gsg_guides.rst (hunk 5 above)
Comment (divipillai) [reply doc/nrf/gsg_guides.rst:20]: My assumption was we would add both the guides. Thanks for clarifying, Greg.

**Review by peknis — APPROVED** [review]

File: doc/_zoomin/ncs.tags.yml [hunk 6]
```diff
@@ -24,12 +24,12 @@ mapping_topics:
   - nrf/gsg_guides/thingy91_gsg.html: ["nrf91-series", "thingy91", "nrf9160", "nrf52840", "lte",
                                        "prototyping-platforms"]
   - nrf/gsg_guides/nrf7002_gs.html: ["nrf70-series", "nrf7002", "development-kits"]
-  - nrf/gsg_guides/nrf5340_gs.html: ["nrf53-series", "nrf5340", "development-kits"]
   - nrf/gsg_guides/thingy53_gs.html: ["nrf53-series", "thingy53", "prototyping-platforms"]
-  - nrf/gsg_guides/nrf52_gs.html: ["nrf52-series", "development-kits", "nrf52840", "nrf52833",
-                                   "nrf52832", "nrf52820", "nrf52811", "nrf52810", "nrf52805"]
-  - nrf/gsg_guides/gsg_other.html: ["nrf9161", "nrf9151", "development-kits", "npm1300", "nrf54h20",
-                                    "nrf54l15", "nrf7002", "evaluation-kits", "pmic"]
+  - nrf/gsg_guides/gsg_other.html: ["nrf52-series", "nrf52840", "nrf52833",
```
Comment (umapraseeda) [inline doc/_zoomin/ncs.tags.yml:28]: Missing nrf9160, thingy91, thingy53

**Review by divipillai — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 7]
```diff
@@ -1163,6 +1163,8 @@ Documentation
   * The Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
   * The Advanced building procedures page and moved its contents to the :ref:`building` page.
   * nRF70 Series support is upstreamed to Zephyr, hence the documentation is removed from the |NCS|.
+  * The standalone pages for getting started with nRF52 Series and with the nRF5340 DK.
+    These pages have been replaced by the `Quick Start`_ app, which now supports the nRF52 Series devices and the nRF5340 DK.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:1167]:
```suggestion
    These pages have been replaced by the ref:`gsg_guides` page, with details of `Quick Start`_ app that now supports the nRF52 Series devices and the nRF5340 DK.
```

Comment (greg-fer) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:1167]: The phrasing is intentional to point the users to the app and skip the unnecessary pages.

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**